### PR TITLE
CLOUD-677 update naming

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

File: source/RN/Kubernetes-Operator-for-PS-RN2.0.0-alpha.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 2.0.0-alpha
 
-*Percona Distribution for MySQL Operator* 2.0.0-alpha
-==============================================================
+*Percona Distribution for MySQL Operator based on Percona Server for MySQL* 0.1.0-alpha
```
Comment (spron-in) [inline source/RN/Kubernetes-Operator-for-PS-RN2.0.0-alpha.rst:3]: I believe we can omit "-alpha" and keep just 0.1.0.
```suggestion
*Percona Distribution for MySQL Operator based on Percona Server for MySQL* 0.1.0
```

File: source/RN/Kubernetes-Operator-for-PS-RN2.0.0-alpha.rst [hunk 2]
```diff
@@ -13,9 +13,9 @@ administrative/operational actions when needed.
 
 The already existing `Percona Distribution for MySQL Operator <https://www.percona.com/doc/kubernetes-operator-for-pxc/index.html>`_ is based on Percona XtraDB Cluster. It is feature rich and provides virtually-synchronous replication by utilizing Galera Write-Sets. Sync replication ensures data consistency and proved itself useful for critical applications, especially on Kubernetes.
 
-The new *Percona Distribution for MySQL Operator 2* is going to run Percona Server for MySQL and provide both regular asynchronous (with `semi-sync <https://dev.mysql.com/doc/refman/8.0/en/replication-semisync.html>`_ support) and virtually-synchronous replication based on `Group Replication <https://dev.mysql.com/doc/refman/8.0/en/group-replication.html>`_.
+The new *Percona Distribution for MySQL Operator* variant is going to run Percona Server for MySQL and provide both regular asynchronous (with `semi-sync <https://dev.mysql.com/doc/refman/8.0/en/replication-semisync.html>`_ support) and virtually-synchronous replication based on `Group Replication <https://dev.mysql.com/doc/refman/8.0/en/group-replication.html>`_.
```
Comment (spron-in) [inline source/RN/Kubernetes-Operator-for-PS-RN2.0.0-alpha.rst:16]:
```suggestion
The new *Percona Distribution for MySQL Operator* is going to run Percona Server for MySQL and provide both regular asynchronous (with `semi-sync <https://dev.mysql.com/doc/refman/8.0/en/replication-semisync.html>`_ support) and virtually-synchronous replication based on `Group Replication <https://dev.mysql.com/doc/refman/8.0/en/group-replication.html>`_.
```

File: source/RN/Kubernetes-Operator-for-PS-RN2.0.0-alpha.rst [hunk 3]
```diff
@@ -13,9 +13,9 @@ administrative/operational actions when needed.
...
 
-**Version 2.0.0-alpha of the Percona Distribution for MySQL Operator is a tech preview release and it is not recommended for production environments.**
+**Version 0.1.0-alpha of the Percona Distribution for MySQL Operator based on Percona Server for MySQL is a tech preview release and it is not recommended for production environments.**
```
Comment (spron-in) [inline source/RN/Kubernetes-Operator-for-PS-RN2.0.0-alpha.rst:18]:
```suggestion
**Version 0.1.0 of the Percona Distribution for MySQL Operator based on Percona Server for MySQL is a tech preview release and it is not recommended for production environments.**
```

**Comment by egegunes** [comment]
@fiowro @spron-in We need to change names in README too

File: source/conf.py [hunk 4]
```diff
@@ -33,17 +33,17 @@
...
 copyright = u'Percona LLC and/or its affiliates 2009 - 2022'
 
 # the short X.Y version
-version = '2.0.0-alpha'
+version = '0.1.0'
 # the full version including alpha/beta/rc tags.
-release = '2.0.0-alpha'
+release = '0.1.0'
 # the PS 5.7 and PS 8.0 recommended versions to be used in docs
 ps57recommended = '5.7.35'
```
Comment (hors) [inline source/conf.py:44]: we do not support 5.7 at all for PS operator

**Review by hors — APPROVED** [review]

**Pull request merged**